_data/projects/Arachnida.yml

```diff
@@ -0,0 +1,17 @@
+name: Arachnida
+desc: Simple web interface to pilot web-crawlers
+site: https://github.com/guillim/Arachnida
+tags:
+- Javascript
+- Meteor
+- crawler
+- scrapping
+- scrapper
+- upforgrabs
+- contributor
+- opensource
+- js
+- Headlesschrome
+upforgrabs:
+  name: Help wanted
```

Comment:
:art: minor thing - the label you are using is `help wanted` (no capital H)

Reply:
yes you're right